orchest-sdk/python/orchest/utils.py

```diff
@@ -30,7 +46,11 @@ def get_step_uuid(pipeline: Pipeline) -> str:
     # Enterprise Gateway.
     kernel_id = os.environ.get("KERNEL_ID")
     if kernel_id is None:
-        raise StepUUIDResolveError('Environment variable "KERNEL_ID" not present.')
+        # Assuming since KERNEL_ID is missing that we're running in
+        # an environment shell. Print environment variable setter hint.
+        pretty_print_env_var_suggestion(pipeline)
+
+        raise StepUUIDResolveError("Could not resolve the pipeline step UUID.")
```

Comment:
Given that the error is called `StepUUIDResolveError` I think it is already clear that the "pipeline step UUID couldn't be resolved". Therefore, I think this gives us the opportunity to use the message we can pass with the error to better explain what happened, e.g.:

```suggestion
        raise StepUUIDResolveError(
            'Environment variable "STEP_UUID" has to be explicitly passed when'
            ' executing Pipeline Steps in Environment shells.'
        )
```